cpp/benchmarks/reduction/scan.cpp

```diff
@@ -34,11 +36,20 @@ static void BM_reduction_scan(benchmark::State& state, bool include_nulls)
   auto const column = create_random_column(dtype, row_count{n_rows});
   if (!include_nulls) column->set_null_mask(rmm::device_buffer{}, 0);
 
+  int64_t result_size = 0;
   for (auto _ : state) {
-    cuda_event_timer timer(state, true);
-    auto result = cudf::scan(
-      *column, *cudf::make_min_aggregation<cudf::scan_aggregation>(), cudf::scan_type::INCLUSIVE);
+    std::unique_ptr<cudf::column> result = nullptr;
+    {
+      cuda_event_timer timer(state, true);
+      result = cudf::scan(
+        *column, *cudf::make_min_aggregation<cudf::scan_aggregation>(), cudf::scan_type::INCLUSIVE);
+    }
+    result_size = estimate_size(result->view());
```

Comment:
This is another place where we do not want to accumulate the result size inside the for-loop.

Reply:
Good catch, fixed!